tshirts/Tshirts.java

```diff
@@ -0,0 +1,21 @@
+public class Tshirts {
+    static String shirtSize(int cms) {
+        if(cms < 38) {
+            return "S";
+        } else if(cms > 38 && cms < 42) {
+            return "M";
+        } else {
+            return "L";
+        }
+    }
+
+    public static void main(String[] args) { 
+        assert(shirtSize(37) == "S");
+        assert(shirtSize(0)  == "M"): "Invalid size ,Size should be non-zero positive value";
+        assert(shirtSize(38) == "M"): "Size is not available";

```

Comment:
As per the code size 38 falls under 'L'.
Either define the test case such that the expected outcome is asserted, or correct the implementation to behave as explicitly defined by the test cases.

Values such as 42 also need to be tested